### PR TITLE
replace Travis badge with Github Actions

### PR DESCRIPTION
**Pull request by bvreede**
Forgot this part in #80 :)

Seeing what the badge actually looks like:

maybe it's a good idea to change the name of the action? Any ideas?

### CODE REVIEW

**Comment by bvreede** [comment]
That said, renaming the workflow simply increases the number of items in actions, and I have not yet found a way to delete them >.<...

So maybe keep it for now?

File: README.rst [hunk 1]
```diff
@@ -8,9 +8,9 @@ era5cli
    :target: http://era5cli.readthedocs.io/en/latest/?badge=latest
    :alt: Documentation Status
 
-.. image:: https://travis-ci.com/eWaterCycle/era5cli.svg?branch=master
-   :target: https://travis-ci.com/eWaterCycle/era5cli
-   :alt: Travis Build
+.. image:: https://github.com/eWaterCycle/era5cli/actions/workflows/test_codecov.yml/badge.svg
+   :target: actions/workflows/test_codecov.yml
```
Comment (stefsmeets) [inline README.rst:12]: This link gives me a 404, maybe it should be like this?

```suggestion
   :target: https://github.com/eWaterCycle/era5cli/actions/workflows/test_codecov.yml
```

**Review by stefsmeets — COMMENTED** [review]
Hi @bvreede , can you check the link target?

**Comment by bvreede** [comment]
Thanks — not sure why I didn't spot that before, much appreciate your catch!

**Review by stefsmeets — APPROVED** [review]
Looks good to me! 🚀

**Pull request merged**